src/components/DaoCreator/index.tsx

```diff
@@ -48,6 +48,7 @@ class DaoCreator extends React.Component<IProps> {
             return await getWeb3Provider();
           }}
           noDAOstackLogo
+          redirectURL="https://alchemy-xdai.daostack.io"
```

Comment:
This does not need to use `targetedNetwork`, can just use `${window.location.protocol}//${window.location.host}` or `process.env.BASE_URL` or some derivation of `RouteComponentProps.location`.  `process.env.BASE_URL` seems easiest.  Whatever the choice, I think the change should be done in this PR.

Reply:
You call sir! @dkent600

Reply:
@cbrzn Cool.  Did you test this?  If yes and it works, I'll approve the PR

Reply:
Yup yup @dkent600 - Would if awesome if you can test locally too :-D. I will mark this conv as resolved :+1: